lib/takeaway.rb

```diff
@@ -0,0 +1,37 @@
+class Takeaway
+
+  attr_reader :basket
+
+  def initialize
+    @menu = {}
+    @basket = {}
+    @total = 0
+  end
+
+  def add_dish(name, price)
+    @menu.store(name, price)
+  end
+
+  def show_menu
+    @menu.to_s
+  end
+
+  def order(name, quantity = 1)
+    @basket.store(name, quantity)
+    puts "#{quantity}x #{name}(s) added to your basket."
+  end
+
+  def total
+    @total = 0
+    @basket.each { |name, quantity|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)